futaba/cogs/moderation/core.py

```diff
@@ -439,3 +439,72 @@ async def unban(self, ctx, user: UserConv, *, reason: str):
             raise ManualCheckFailure(
                 content="I don't have permission to unban this user"
             )
+
+    @commands.command(name="gist", aliases=["msgupload"])
+    @commands.guild_only()
+    async def upload_message(self, ctx, *messages: MessageConv):
+        """
+        Concatenates the range of messages and upload to a gist.
+        A link to the gist is posted after a successful upload.
+        """
+
+        if not self.bot.config.gist_oauth_token:
+            raise CommandFailed(content="The gist oauth token is not configured.")
+
+        messages_content = "\n".join(str(message.content) for message in messages)
```

Comment:
If this allows multiple messages it would be a good idea to add the username to this so we know who said what and that in the gist